relay/network_graph/dijkstra_weighted.py

```diff
@@ -87,12 +20,14 @@ def find_path(G, source, target, get_fee, value, max_hops=None, max_fees=None):
         (n, d, _, v) = pop(fringe)
         if v in dist:
             continue  # already searched this node.
+        if v == ignore:
+            continue
         dist[v] = d
         if v == target:
             break
 
         for u, e in G_succ[v].items():
-            cost = get_fee(v, u, e, d)
+            cost = get_fee(v, u, e, d)  # fee of transferring from u to v
```

Comment:
Is that actually the fee from u to v? Than we should fix the get_fee function.

Reply:
Yes, since in the application for a path in reverse, for the first iteration, we have "v = destination", u is one neighbor of destination. We then need the fee of transferring from u to v.